ballerina/client.bal

```diff
@@ -155,29 +155,29 @@ public isolated client class Client {
 
     # Acknowledges one or several received messages.
     # ```ballerina
-    # check rabbitmqClient->basicAck(<message>);
+    # check rabbitmqClient->basicAck(<message|deliveryTag>);
     # ```
     #
-    # + message - The message to be acknowledged
+    # + ackTarget - The message to be acknowledged or the delivery tag of the message
     # + multiple - Set to `true` to acknowledge all messages up to and including the called on message and
     #              `false` to acknowledge just the called on message
     # + return - A `rabbitmq:Error` if an I/O error occurred or else `()`
-    isolated remote function basicAck(AnydataMessage message, boolean multiple = false) returns Error? =
+    isolated remote function basicAck(AnydataMessage|int ackTarget, boolean multiple = false) returns Error? =
```

Comment:
As per my understanding this is a breaking change and since we have not released rabbitmq under SL U12, we could do a minor version bump on top of U12. So, shall we update the package version in this PR as well.